client/init.c

```diff
@@ -136,7 +141,13 @@ int parse_startup_parameters(int argc, char **argv, struct startup_parameters *p
 			}
 			continue;
 		}
-		if(ARG_EQUAL(argv[i], "-f", "")) { /* configuration file */
+        // move -t , -disable-refresh befor -f
```

Comment:
Why?

Reply:
./xdag  -t -f pool_1.config -d -disable-refresh
this is a bug, when we use -f to start xdag pools, befause after -f return 0, can trunc -d -disable-refresh args